intelmq/bots/experts/reverse_dns/expert.py

```diff
@@ -61,8 +64,10 @@ def process(self):
                         result = None
                         raise ValueError
                 except (dns.exception.DNSException, ValueError) as e:
-                    if isinstance(e, dns.resolver.NXDOMAIN):
-                        continue
+                    # Set longer TTL for 'DNS query name does not exist' error
+                    ttl = 3600 if isinstance(e, dns.resolver.NXDOMAIN) else 60
```

Comment:
Why are you using another TTL for NXDOMAIN as for regular responses?

And I'd like to make the TTL in case of other errors configurable too, e.g. with `cache_ttl_invalid_response = 60` for the reverse dns.